CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 5.8.1
```

Comment:
Fine by me 👌 
Perhaps throw a note on versioning in the README so the convention is explicitly stated somewhere?

Reply:
Done in https://github.com/Intellection/docker-elasticsearch-curator/pull/4/commits/ac15e6900e142915ddec8527f4f3e0a843e62d58.